pkg/operator/ceph/cluster/cephstatus.go

```diff
@@ -163,6 +165,20 @@ func toCustomResourceStatus(currentStatus cephv1.ClusterStatus, newStatus *cephc
 			Message:  message.Summary.Message,
 		}
 	}
+
+	if newStatus.PgMap.TotalBytes != 0 {
+		s.Capacity.TotalBytes = newStatus.PgMap.TotalBytes
+		s.Capacity.UsedBytes = newStatus.PgMap.UsedBytes
+		s.Capacity.AvailableBytes = newStatus.PgMap.AvailableBytes
+		s.Capacity.LastUpdated = formatTime(time.Now().UTC())
+		previousCheck = true
+	}
+
+	if previousCheck && newStatus.PgMap.TotalBytes == 0 {
```

Comment:
I don't think we need a previousCheck var, we can just check for a non-zero value on the current capacity.
```suggestion
	if currentStatus.CephStatus.Capacity.TotalBytes != 0 && newStatus.PgMap.TotalBytes == 0 {
```